web3j-ext/web3j-ext/src/main/java/org/web3j/protocol/klaytn/Web3j.java

```diff
@@ -15,9 +15,10 @@
 import org.web3j.protocol.klaytn.core.NetApi;
 import org.web3j.protocol.klaytn.core.PersonalApi;
 import org.web3j.protocol.klaytn.core.TxpoolApi;
+import org.web3j.protocol.klaytn.core.KaiaApi;
```

Comment:
@JayChoi1736 We need to deploy the web3rpc-java first, right?

Reply:
yes, right.
we need to add commit for build.gradle fix, too